src/main/java/com/sperasoft/passportapi/controller/PersonController.java

```diff
@@ -0,0 +1,39 @@
+package com.sperasoft.passportapi.controller;
+
+import com.sperasoft.passportapi.controller.dto.PersonRequest;
+import com.sperasoft.passportapi.controller.dto.PersonResponse;
+import com.sperasoft.passportapi.service.PersonServiceImpl;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@RequestMapping("/person")
+@RequiredArgsConstructor
+public class PersonController {
+
+    private final PersonServiceImpl personService;
+
+    @PostMapping
+    public PersonResponse createPerson(@RequestBody @Valid PersonRequest person) {
+        return personService.addPerson(person);
```

Comment:
PersonService should not be aware about controller level DTOs.

Reply:
- [x]  fixed